RPs to go on-chain on 2023-06-12 (#2301)
makerdao/community: governance/polls/Ratification Poll - MIP Amendment Subproposal (MIP102c2-SP9) - June 12, 2023.md

---
title: Ratification Poll for MIP Amendment Subproposal (MIP102c2-SP9) - June 12, 2023

> summary: This subproposal incorporates the Bug Bounty Programa (originally MIP64) into MIP107.

Reviewer: ```suggestion
summary: This subproposal incorporates the Bug Bounty Program (originally MIP64) into MIP107.
```

discussion_link: https://forum.makerdao.com/t/mip102c2-sp9-mip-amendment-subproposal/20640
parameters:
    input_format:
        type: single-choice
        abstain: [0]
    victory_conditions:
        - {
            type: 'and',
            conditions: [
                { type : plurality },
                { type : comparison, comparator : '>=', value: 10000 }
            ]
        }
        - {type : default, value : 2 }
    result_display: single-vote-breakdown
version: v2.0.0
options:
   0: Abstain
   1: Yes
   2: No
start_date: 2023-06-12T16:00:00
end_date: 2023-06-26T16:00:00
---
# Ratification Poll for MIP Amendment Subproposal (MIP102c2-SP9) - June 12, 2023

> The Arbitration Facilitators have placed a ratification poll into the [voting system](https://vote.makerdao.com/polling) as part of the responsibilities defined in [MIP51](https://mips.makerdao.com/mips/details/MIP51). This Governance [Poll](https://manual.makerdao.com/governance/governance-cycle/weekly-governance-cycle#weekly-governance-cycle-definitions-mip16c1) will be active for fourteen days beginning on Monday, June 12 at 16:00 UTC.

Reviewer: ```suggestion
The Governance Facilitator has placed a ratification poll into the [voting system](https://vote.makerdao.com/polling) as part of the responsibilities defined in [MIP51](https://mips.makerdao.com/mips/details/MIP51). This Governance [Poll](https://manual.makerdao.com/governance/governance-cycle/weekly-governance-cycle#weekly-governance-cycle-definitions-mip16c1) will be active for fourteen days beginning on Monday, June 12 at 16:00 UTC.
```

Reviewer: Should the last link here go to the monthly governance cycle page rather than the weekly?

## Review

The community may vote in this poll to express support or opposition to MIP102c2-SP9 being accepted and implemented in the Maker Protocol.

A brief summary of this proposal has been provided by the MIP Author and is shown below:

*This subproposal ensures continuity of the bug bounty program (i.e., MIP64) by incorporating it into the Scopes framework.*

Please review the links below to inform your position on this proposal before voting.
* [Canonical Proposal Version](https://github.com/makerdao/mips/blob/98a7c6f6a009fffa692ceaa7c04f0b4a8da750df/MIP102/MIP102c2-Subproposals/MIP102c2-SP9.md)
* [Latest Proposal Version](https://mips.makerdao.com/mips/details/MIP102c2SP9)
* [Proposal Discussion Thread](https://forum.makerdao.com/t/mip102c2-sp9-mip-amendment-subproposal/20640)
* [Amendment Pull Request](https://github.com/makerdao/mips/pull/898)

## Outcomes

This poll implements a **Minimum Positive Participation** value. The Minimum Positive Participation is currently set to **10,000 MKR**.

**If the votes for the 'Yes' option exceed the votes for the 'No' option AND the votes for the 'Yes' option exceed 10,000 MKR, then the following actions will be taken:**

> * The Arbitration Facilitators will confirm its passage by marking the proposal **Accepted** on the [MIPs portal](https://mips.makerdao.com/mips/list) and across all other relevant governance mediums.

Reviewer: ```suggestion
* The Governance Facilitator will confirm its passage by marking the proposal **Accepted** on the [MIPs portal](https://mips.makerdao.com/mips/list) and across all other relevant governance mediums.
```

> * Any further work required to implement the proposal will be tasked to the relevant [Ecosystem Actors](https://mips.makerdao.com/mips/details/MIP101#7-professional-actors).

Reviewer: Link is semi-dead
